src/client/sandbox/event/hover.js

```diff
@@ -56,6 +58,14 @@ export default class HoverSandbox extends SandboxBase {
         return jointParent;
     }
 
+    _onHover ({ target, clientX, clientY }) {
+        const isIframeContent = domUtils.getTagName(target) === 'iframe' && positionUtils.isPositionInsideElement(target, clientX, clientY);
```

Comment:
1)Convert expression
 `domUtils.getTagName(target) === 'iframe' && positionUtils.isPositionInsideElement(target, clientX, clientY)` 
to utility function `isPositionInsideIframe(el, {x, y})` and move this to the `src/client/utils/position.js` file.

2)Merge `onHover` and `_hover` function together. And rewrite using fast forward paradigm
```js
_hover() {
 const isPositionIsInIframe = postionUtils.isInIframe(el, x, y);
 const shouldMarkElement = browserUtils.isIE && isPositionIsInIframe;

if (!shouldMarkElement)
  return;

....
}

```

Reply:
1) ok
2) I've separated them intentionally, because `_onHover` method takes `x,y` params while `_hover` is just api method without any extra params. Besides `_hover` is used in our tests